docs/components/tuner/evaluation.md

```diff
@@ -0,0 +1,72 @@
+# Evaluation
+
+Information retrieval metrics like NDCG and Reciprocal Rank can be calculated using the included 
+{class}`~finetuner.tuner.evaluation.Evaluator` component. The evaluator can be used
+[standalone](#using-the-evaluator)
+or integrated in the training loop via the [evaluation callback](#using-the-evaluation-callback).
+
+
+## Using the evaluator
+
+The evaluator can be used standalone in order to compute the evaluation metrics on a sequence
+of documents:
+```python
+from finetuner.tuner.evaluation import Evaluator
+from docarray import DocumentArray
+
+query_data = DocumentArray(...)
+index_data = DocumentArray(...)
+embed_model = ...
+
+evaluator = Evaluator(query_data, index_data, embed_model)
+
+metrics = evaluator.evaluate()
```

Comment:
Is it possible that we provide an actual working example here? I think it would be more informative

Reply:
sure I'll add an actual example instead of dots